lib/src/splash.dart

```diff
@@ -53,11 +50,17 @@ class _FlameSplashScreenState extends State<FlameSplashScreen> {
     controller.setup(steps.length, onFinish);
   }
 
+  @override
+  void reassemble() {
+    // TODO: implement reassemble
+    super.reassemble();
+  }
+
   void computeSteps() {
     steps = [
-      if (widget.showBefore != null) widget.showBefore,
+      if (widget.showBefore != null) widget.showBefore!,
       widget.theme.logoBuilder,
-      if (widget.showAfter != null) widget.showAfter
+      if (widget.showAfter != null) widget.showAfter!
```

Comment:
nit: save showBefore and After to variables so dart can smart cast

Reply:
this would affect these inline ifs and make things longer